CHANGELOG.md

```diff
@@ -118,6 +122,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Initial release
 
 [Unreleased]: https://github.com/GoogleCloudPlatform/functions-framework-python/compare/v2.3.0...HEAD
```

Comment:
I believe this needs to be modified too.

Reply:
Done